src/core/core_structs/feedforward.jl

```diff
@@ -1,6 +1,6 @@
-abstract type FeedForwardModel end
+abstract type AbstractFeedForwardAffect end
```

Comment:
Should this be `Effect` instead of `Affect`?

Reply:
https://www.grammarly.com/blog/affect-vs-effect/

Reply:
Affect is usually a verb, and it means to impact or change. Effect is usually a noun, an effect is the result of a change.

Reply:
When I'm thinking about this I'm hearing myself say this: "When you apply this `FeedForwardEffect` interaction on `n` and `m` `Stage`s, so and so values from `Stage` `n` are fed to so and so fields in `Stage` `m`.".